src/app/f2f/controllers/landingPage.test.js

```diff
@@ -0,0 +1,35 @@
+const BaseController = require("hmpo-form-wizard").Controller;
+const { expect } = require("chai");
+const LandingPageController = require('./landingPage');
+
+describe("LandingPage", () => {
+  let landingPageController;
+
+  let req;
+  let res;
+  let next;
+
+  beforeEach(() => {
+    const setup = setupDefaultMocks();
+    req = setup.req;
+    res = setup.res;
+    next = setup.next;
+
+    landingPageController = new LandingPageController({ route: "/test" });
+  });
+
+  it("should be an instance of BaseController", () => {
+    expect(landingPageController).to.be.an.instanceOf(BaseController);
+  });
+
+  describe("#saveValues", () => {
+    it("should save the values to the sessionModel", async () => {
+      req.session.tokenId = "123abc"
+
+      await landingPageController.saveValues(req, res, next);
+      expect(next).to.have.been.calledOnce;
```

Comment:
There's quite a lot of other logic in the `saveValues` function (eg setting `isThinFileUser` to false, fetching the session config, etc.) is there a reason that we haven't tested it?

Reply:
It's mainly time constraints honestly. I picked the easiest things to test to get the coverage over 60% (it should be around 70% now)